src/css/fluid-covid-map-viz.css

```diff
@@ -0,0 +1,337 @@
+body {
+    margin: 10px;
+    font-family: sans-serif;
+}
+
+.fld-mapviz-map-holder {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+/*    justify-content: center;*/
+    align-items: center;
+}
+
+.flc-query-holder > * {
+     margin-top: 4px;
+     margin-bottom: 7px;
+}
+
+.fld-mapviz-query {
+    width: 700px;
+    height: 2rem;
+    border-radius: 20px;
```

Comment:
Use `rem` instead of `px` whenever applicable. This comment applies to other css code too.